### DISCUSSION TITLE
Hovering over options should delay the start of the map

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#30084] [Ideas] [answered]
Upon selecting and starting a difficulty to play, one can see mulpiple "windows" containing options at the right side and sometimes a window on the left as well (for example for the loved message). There is also a bar on top.
![osu_2024-10-01_19-55-18](https://github.com/user-attachments/assets/1628b9d7-7805-4607-925e-0ea56380810d)

My suggestion is that if the player is hovering over one of those areas that the beatmap won't start, so that the player has enough time to read everything. This would be especially good for new players when they see those settings for the first time.

**Comment by bdach** [comment, marked as answer]
It already does this though.

**Reply by peppy** [reply]
The exception is if you leave the mouse still for too long. Maybe that delay needs to be longer, but it does exist for a reason (if a user just leaves their cursor there expecting things to start and they don't).
